browser/component_updater/brave_component_updater_configurator.cc

```diff
@@ -62,7 +62,14 @@ BraveConfigurator::BraveConfigurator(
               [](PrefService* pref_service) { return pref_service; },
               pref_service),
           nullptr)),
-      url_loader_factory_(std::move(url_loader_factory)) {}
+      url_loader_factory_(std::move(url_loader_factory)) {
+  base::FilePath path;
+  bool result = base::PathService::Get(chrome::DIR_USER_DATA, &path);
+  crx_cache_ = base::MakeRefCounted<update_client::CrxCache>(
```

Comment:
why are we duplicating this from upstream?

Reply:
I think most likely, but yes let's create a follow-up to look at it

Reply:
Makes sense, filed https://github.com/brave/brave-browser/issues/46626 for this.